source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -470,7 +525,7 @@ world-enabled
 
 ------------------------------------------------------------------------------------------------------------
 
-This config was generated using SpongeForge build 3442 (with Forge 2705), SpongeAPI version 7.1.0:
+This config was generated using SpongeForge recommendation 5 (SpongeForge 3554 with Forge 2768), SpongeAPI version 7.1:
```

Comment:
Is there a better way to refer to the recommended build 5?

Reply:
That *should* be `SpongeForge recommended build 1.12.2-2768-7.1.5`, (which is equivalent to `release candidate 12.2-2768-7.1.5-RC3554`).

Reply:
Uh, I don't actually remember changing this part. I'll change it to 7.1.5.